lib/inspec/resources/sys_info.rb

```diff
@@ -26,5 +26,29 @@ def hostname
         skip_resource "The `sys_info.hostname` resource is not supported on your OS yet."
       end
     end
+
+    # returns the Manufacturer of the local system
+    def manufacturer
+      os = inspec.os
+      if os.linux? || os.darwin?
+        inspec.command("dmidecode | grep 'Vendor :'").split(": ").last.chomp
```

Comment:
Hrm, `dmidecode` is not available on my MacOS (10.14.4 Mojave). Maybe you have it installed via homebrew?

Reply:
I have no OSX at hand, but according to https://serverfault.com/questions/112711/how-can-i-get-cpu-count-and-total-ram-from-the-os-x-command-line/397809#397809, `system_profiler` should also work. Added it in the latest commit.